### ISSUE TITLE
vminfo Error: Unable to connect to system D-Bus

### ISSUE THREAD

**Issue by rillig** [docker/kitematic#3645]
### Expected behavior

When starting Kitematic, it boots the Docker VM.

### Actual behavior

![image](https://user-images.githubusercontent.com/3233724/37251679-d73db404-2514-11e8-8fb2-7f4c75b79680.png)

The error messages, after replacing `,(?! )` with `,\n`, are:

```plain
Command failed: C:\Program Files\Docker Toolbox\docker-machine.exe -D start default,
Docker Machine Version: 0.13.0, build 9ba6da9,
Found binary path at C:\Program Files\Docker Toolbox\docker-machine.exe,
Launching plugin server for driver virtualbox,
Plugin server listening at address 127.0.0.1:54708,
() Calling .GetVersion,
Using API Version 1,
() Calling .SetConfigRaw,
() Calling .GetMachineName,
command=start machine=default,
(default) Calling .GetState,
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe showvminfo default --machinereadable,
(default) DBG | STDOUT:,
(default) DBG | {,
(default) DBG | name="default",
(default) DBG | groups="/",
(default) DBG | ostype="Linux 2.6 / 3.x / 4.x (64-bit)",
(default) DBG | UUID="1cf45276-9a14-4c1b-9dfd-045a2367d801",
(default) DBG | CfgFile="C:\\Users\\rillig\\.docker\\machine\\machines\\default\\default\\default.vbox",
(default) DBG | SnapFldr="C:\\Users\\rillig\\.docker\\machine\\machines\\default\\default\\Snapshots",
(default) DBG | LogFldr="C:\\Users\\rillig\\.docker\\machine\\machines\\default\\default\\Logs",
(default) DBG | hardwareuuid="1cf45276-9a14-4c1b-9dfd-045a2367d801",
(default) DBG | memory=1024,
(default) DBG | pagefusion="off",
(default) DBG | vram=8,
(default) DBG | cpuexecutioncap=100,
(default) DBG | hpet="on",
(default) DBG | chipset="piix3",
(default) DBG | firmware="BIOS",
(default) DBG | cpus=1,
(default) DBG | pae="on",
(default) DBG | longmode="on",
(default) DBG | triplefaultreset="off",
(default) DBG | apic="on",
(default) DBG | x2apic="off",
(default) DBG | cpuid-portability-level=0,
(default) DBG | bootmenu="disabled",
(default) DBG | boot1="dvd",
(default) DBG | boot2="dvd",
(default) DBG | boot3="disk",
(default) DBG | boot4="none",
(default) DBG | acpi="on",
(default) DBG | ioapic="on",
(default) DBG | biosapic="apic",
(default) DBG | biossystemtimeoffset=0,
(default) DBG | rtcuseutc="on",
(default) DBG | hwvirtex="on",
(default) DBG | nestedpaging="on",
(default) DBG | largepages="on",
(default) DBG | vtxvpid="on",
(default) DBG | vtxux="on",
(default) DBG | paravirtprovider="default",
(default) DBG | effparavirtprovider="kvm",
(default) DBG | VMState="aborted",
(default) DBG | VMStateChangeTime="2018-03-11T00:32:00.000000000",
(default) DBG | monitorcount=1,
(default) DBG | accelerate3d="off",
(default) DBG | accelerate2dvideo="off",
(default) DBG | teleporterenabled="off",
(default) DBG | teleporterport=0,
(default) DBG | teleporteraddress="",
(default) DBG | teleporterpassword="",
(default) DBG | tracing-enabled="off",
(default) DBG | tracing-allow-vm-access="off",
(default) DBG | tracing-config="",
(default) DBG | autostart-enabled="off",
(default) DBG | autostart-delay=0,
(default) DBG | defaultfrontend="",
(default) DBG | storagecontrollername0="SATA",
(default) DBG | storagecontrollertype0="IntelAhci",
(default) DBG | storagecontrollerinstance0="0",
(default) DBG | storagecontrollermaxportcount0="30",
(default) DBG | storagecontrollerportcount0="30",
(default) DBG | storagecontrollerbootable0="on",
(default) DBG | "SATA-0-0"="C:\Users\rillig\.docker\machine\machines\default\boot2docker.iso",
(default) DBG | "SATA-ImageUUID-0-0"="3dd1cf6a-3f97-47fd-9cbd-6f79769fd951",
(default) DBG | "SATA-tempeject"="off",
(default) DBG | "SATA-IsEjected"="off",
(default) DBG | "SATA-1-0"="C:\Users\rillig\.docker\machine\machines\default\disk.vmdk",
(default) Calling .Start,
(default) DBG | "SATA-ImageUUID-1-0"="f672c646-c992-4a91-aaca-36fcd5b83aa0",
(default) DBG | "SATA-2-0"="none",
(default) DBG | "SATA-3-0"="none",
(default) DBG | "SATA-4-0"="none",
(default) DBG | "SATA-5-0"="none",
(default) DBG | "SATA-6-0"="none",
(default) DBG | "SATA-7-0"="none",
(default) DBG | "SATA-8-0"="none",
(default) DBG | "SATA-9-0"="none",
(default) DBG | "SATA-10-0"="none",
(default) DBG | "SATA-11-0"="none",
(default) DBG | "SATA-12-0"="none",
(default) DBG | "SATA-13-0"="none",
(default) DBG | "SATA-14-0"="none",
(default) DBG | "SATA-15-0"="none",
(default) DBG | "SATA-16-0"="none",
(default) DBG | "SATA-17-0"="none",
(default) DBG | "SATA-18-0"="none",
(default) DBG | "SATA-19-0"="none",
(default) DBG | "SATA-20-0"="none",
(default) DBG | "SATA-21-0"="none",
(default) DBG | "SATA-22-0"="none",
(default) DBG | "SATA-23-0"="none",
(default) DBG | "SATA-24-0"="none",
(default) DBG | "SATA-25-0"="none",
(default) DBG | "SATA-26-0"="none",
(default) DBG | "SATA-27-0"="none",
(default) DBG | "SATA-28-0"="none",
(default) DBG | "SATA-29-0"="none",
(default) DBG | natnet1="nat",
(default) DBG | macaddress1="0800276A9A85",
(default) DBG | cableconnected1="on",
(default) DBG | nic1="nat",
(default) DBG | nictype1="82540EM",
(default) DBG | nicspeed1="0",
(default) DBG | mtu="0",
(default) DBG | sockSnd="64",
(default) DBG | sockRcv="64",
(default) DBG | tcpWndSnd="64",
(default) DBG | tcpWndRcv="64",
(default) DBG | Forwarding(0)="ssh,
tcp,
127.0.0.1,
58634,
,
22",
(default) DBG | hostonlyadapter2="VirtualBox Host-Only Ethernet Adapter #2",
(default) DBG | macaddress2="080027F9FF44",
(default) DBG | cableconnected2="on",
(default) DBG | nic2="hostonly",
(default) DBG | nictype2="82540EM",
(default) DBG | nicspeed2="0",
(default) DBG | nic3="none",
(default) DBG | nic4="none",
(default) DBG | nic5="none",
(default) DBG | nic6="none",
(default) DBG | nic7="none",
(default) DBG | nic8="none",
(default) DBG | hidpointing="ps2mouse",
(default) DBG | hidkeyboard="ps2kbd",
(default) DBG | uart1="off",
(default) DBG | uart2="off",
(default) DBG | uart3="off",
(default) DBG | uart4="off",
(default) DBG | lpt1="off",
(default) DBG | lpt2="off",
(default) DBG | audio="dsound",
(default) DBG | audio_in="false",
(default) DBG | audio_out="false",
(default) DBG | clipboard="disabled",
(default) DBG | draganddrop="disabled",
(default) DBG | vrde="off",
(default) DBG | usb="off",
(default) DBG | ehci="off",
(default) DBG | xhci="off",
(default) DBG | SharedFolderNameMachineMapping1="c/Users",
(default) DBG | SharedFolderPathMachineMapping1="\\?\c:\Users",
(default) DBG | videocap="off",
(default) DBG | videocap_audio="off",
(default) DBG | videocapscreens=0,
(default) DBG | videocapfile="C:\Users\rillig\.docker\machine\machines\default\default\default.webm",
(default) DBG | videocapres=1024x768,
(default) DBG | videocaprate=512,
(default) DBG | videocapfps=25,
(default) DBG | videocapopts=ac_enabled=false,
(default) DBG | GuestMemoryBalloon=0,
(default) DBG | },
(default) DBG | STDERR:,
(default) DBG | {,
(default) DBG | },
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe showvminfo default --machinereadable,
(default) DBG | STDOUT:,
(default) DBG | {,
(default) DBG | name="default",
(default) DBG | groups="/",
(default) DBG | ostype="Linux 2.6 / 3.x / 4.x (64-bit)",
(default) DBG | UUID="1cf45276-9a14-4c1b-9dfd-045a2367d801",
(default) DBG | CfgFile="C:\\Users\\rillig\\.docker\\machine\\machines\\default\\default\\default.vbox",
(default) DBG | SnapFldr="C:\\Users\\rillig\\.docker\\machine\\machines\\default\\default\\Snapshots",
(default) DBG | LogFldr="C:\\Users\\rillig\\.docker\\machine\\machines\\default\\default\\Logs",
(default) DBG | hardwareuuid="1cf45276-9a14-4c1b-9dfd-045a2367d801",
(default) DBG | memory=1024,
(default) DBG | pagefusion="off",
(default) DBG | vram=8,
(default) DBG | cpuexecutioncap=100,
(default) DBG | hpet="on",
(default) DBG | chipset="piix3",
(default) DBG | firmware="BIOS",
(default) DBG | cpus=1,
(default) DBG | pae="on",
(default) DBG | longmode="on",
(default) DBG | triplefaultreset="off",
(default) DBG | apic="on",
(default) DBG | x2apic="off",
(default) DBG | cpuid-portability-level=0,
(default) DBG | bootmenu="disabled",
(default) DBG | boot1="dvd",
(default) DBG | boot2="dvd",
(default) DBG | boot3="disk",
(default) DBG | boot4="none",
(default) DBG | acpi="on",
(default) DBG | ioapic="on",
(default) DBG | biosapic="apic",
(default) DBG | biossystemtimeoffset=0,
(default) DBG | rtcuseutc="on",
(default) DBG | hwvirtex="on",
(default) DBG | nestedpaging="on",
(default) DBG | largepages="on",
(default) DBG | vtxvpid="on",
(default) DBG | vtxux="on",
(default) DBG | paravirtprovider="default",
(default) DBG | effparavirtprovider="kvm",
(default) DBG | VMState="aborted",
(default) DBG | VMStateChangeTime="2018-03-11T00:32:00.000000000",
(default) DBG | monitorcount=1,
(default) DBG | accelerate3d="off",
(default) DBG | accelerate2dvideo="off",
(default) DBG | teleporterenabled="off",
(default) DBG | teleporterport=0,
(default) DBG | teleporteraddress="",
(default) DBG | teleporterpassword="",
(default) DBG | tracing-enabled="off",
(default) DBG | tracing-allow-vm-access="off",
(default) DBG | tracing-config="",
(default) DBG | autostart-enabled="off",
(default) DBG | autostart-delay=0,
(default) DBG | defaultfrontend="",
(default) DBG | storagecontrollername0="SATA",
(default) DBG | storagecontrollertype0="IntelAhci",
(default) DBG | storagecontrollerinstance0="0",
(default) DBG | storagecontrollermaxportcount0="30",
(default) DBG | storagecontrollerportcount0="30",
(default) DBG | storagecontrollerbootable0="on",
(default) DBG | "SATA-0-0"="C:\Users\rillig\.docker\machine\machines\default\boot2docker.iso",
(default) DBG | "SATA-ImageUUID-0-0"="3dd1cf6a-3f97-47fd-9cbd-6f79769fd951",
(default) DBG | "SATA-tempeject"="off",
(default) DBG | "SATA-IsEjected"="off",
(default) DBG | "SATA-1-0"="C:\Users\rillig\.docker\machine\machines\default\disk.vmdk",
(default) DBG | "SATA-ImageUUID-1-0"="f672c646-c992-4a91-aaca-36fcd5b83aa0",
(default) DBG | "SATA-2-0"="none",
(default) DBG | "SATA-3-0"="none",
(default) DBG | "SATA-4-0"="none",
(default) DBG | "SATA-5-0"="none",
(default) DBG | "SATA-6-0"="none",
(default) DBG | "SATA-7-0"="none",
(default) DBG | "SATA-8-0"="none",
(default) DBG | "SATA-9-0"="none",
(default) DBG | "SATA-10-0"="none",
(default) DBG | "SATA-11-0"="none",
(default) DBG | "SATA-12-0"="none",
(default) DBG | "SATA-13-0"="none",
(default) DBG | "SATA-14-0"="none",
(default) DBG | "SATA-15-0"="none",
(default) DBG | "SATA-16-0"="none",
(default) DBG | "SATA-17-0"="none",
(default) DBG | "SATA-18-0"="none",
(default) DBG | "SATA-19-0"="none",
(default) DBG | "SATA-20-0"="none",
(default) DBG | "SATA-21-0"="none",
(default) DBG | "SATA-22-0"="none",
(default) DBG | "SATA-23-0"="none",
(default) DBG | "SATA-24-0"="none",
(default) DBG | "SATA-25-0"="none",
(default) DBG | "SATA-26-0"="none",
(default) DBG | "SATA-27-0"="none",
(default) DBG | "SATA-28-0"="none",
(default) DBG | "SATA-29-0"="none",
(default) DBG | natnet1="nat",
(default) DBG | macaddress1="0800276A9A85",
(default) DBG | cableconnected1="on",
(default) DBG | nic1="nat",
(default) DBG | nictype1="82540EM",
(default) DBG | nicspeed1="0",
(default) DBG | mtu="0",
(default) DBG | sockSnd="64",
(default) DBG | sockRcv="64",
(default) DBG | tcpWndSnd="64",
(default) DBG | tcpWndRcv="64",
(default) DBG | Forwarding(0)="ssh,
tcp,
127.0.0.1,
58634,
,
22",
(default) DBG | hostonlyadapter2="VirtualBox Host-Only Ethernet Adapter #2",
(default) DBG | macaddress2="080027F9FF44",
(default) DBG | cableconnected2="on",
(default) DBG | nic2="hostonly",
(default) DBG | nictype2="82540EM",
(default) DBG | nicspeed2="0",
(default) DBG | nic3="none",
(default) DBG | nic4="none",
(default) DBG | nic5="none",
(default) DBG | nic6="none",
(default) DBG | nic7="none",
(default) DBG | nic8="none",
(default) DBG | hidpointing="ps2mouse",
(default) DBG | hidkeyboard="ps2kbd",
(default) DBG | uart1="off",
(default) DBG | uart2="off",
(default) DBG | uart3="off",
(default) DBG | uart4="off",
(default) DBG | lpt1="off",
(default) DBG | lpt2="off",
(default) DBG | audio="dsound",
(default) DBG | audio_in="false",
(default) DBG | audio_out="false",
(default) DBG | clipboard="disabled",
(default) DBG | draganddrop="disabled",
(default) DBG | vrde="off",
(default) DBG | usb="off",
(default) DBG | ehci="off",
(default) DBG | xhci="off",
(default) DBG | SharedFolderNameMachineMapping1="c/Users",
(default) DBG | SharedFolderPathMachineMapping1="\\?\c:\Users",
(default) DBG | videocap="off",
(default) DBG | videocap_audio="off",
(default) DBG | videocapscreens=0,
(default) DBG | videocapfile="C:\Users\rillig\.docker\machine\machines\default\default\default.webm",
(default) DBG | videocapres=1024x768,
(default) DBG | videocaprate=512,
(default) DBG | videocapfps=25,
(default) DBG | videocapopts=ac_enabled=false,
(default) DBG | GuestMemoryBalloon=0,
(default) DBG | },
(default) DBG | STDERR:,
(default) DBG | {,
(default) DBG | },
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe list hostonlyifs,
(default) DBG | STDOUT:,
(default) DBG | {,
(default) DBG | Name: VirtualBox Host-Only Ethernet Adapter #2,
(default) DBG | GUID: d919871a-dafe-43ba-9d6c-ddffd058f38d,
(default) DBG | DHCP: Disabled,
(default) DBG | IPAddress: 192.168.99.1,
(default) DBG | NetworkMask: 255.255.255.0,
(default) DBG | IPV6Address: fe80::f08c:4842:d7b0:8760,
(default) DBG | IPV6NetworkMaskPrefixLength: 64,
(default) DBG | HardwareAddress: 0a:00:27:00:00:10,
(default) DBG | MediumType: Ethernet,
(default) DBG | Wireless: No,
(default) DBG | Status: Up,
(default) DBG | VBoxNetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter #2,
(default) DBG | ,
(default) DBG | Name: VirtualBox Host-Only Ethernet Adapter,
(default) DBG | GUID: f29686ae-06ce-46c1-aa63-53139bf06fa3,
(default) DBG | DHCP: Disabled,
(default) DBG | IPAddress: 192.168.56.1,
(default) DBG | NetworkMask: 255.255.255.0,
(default) DBG | IPV6Address: fe80::d42b:57f4:8a96:a072,
(default) DBG | IPV6NetworkMaskPrefixLength: 64,
(default) DBG | HardwareAddress: 0a:00:27:00:00:11,
(default) DBG | MediumType: Ethernet,
(default) DBG | Wireless: No,
(default) DBG | Status: Up,
(default) DBG | VBoxNetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter,
(default) DBG | ,
(default) DBG | },
(default) DBG | STDERR:,
(default) DBG | {,
(default) DBG | },
(default) DBG | Searching for hostonly interface for IPv4: 192.168.99.1 and Mask: ffffff00,
(default) DBG | Found: VirtualBox Host-Only Ethernet Adapter #2,
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe list dhcpservers,
(default) DBG | STDOUT:,
(default) DBG | {,
(default) DBG | NetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter,
(default) DBG | IP: 192.168.56.100,
(default) DBG | NetworkMask: 255.255.255.0,
(default) DBG | lowerIPAddress: 192.168.56.101,
(default) DBG | upperIPAddress: 192.168.56.254,
(default) DBG | Enabled: Yes,
(default) DBG | ,
(default) DBG | NetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter #2,
(default) DBG | IP: 192.168.99.12,
(default) DBG | NetworkMask: 255.255.255.0,
(default) DBG | lowerIPAddress: 192.168.99.100,
(default) DBG | upperIPAddress: 192.168.99.254,
(default) DBG | Enabled: Yes,
(default) DBG | ,
(default) DBG | },
(default) DBG | STDERR:,
(default) DBG | {,
(default) DBG | },
(default) DBG | Removing orphan DHCP servers...,
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe list hostonlyifs,
(default) DBG | STDOUT:,
(default) DBG | {,
(default) DBG | Name: VirtualBox Host-Only Ethernet Adapter #2,
(default) DBG | GUID: d919871a-dafe-43ba-9d6c-ddffd058f38d,
(default) DBG | DHCP: Disabled,
(default) DBG | IPAddress: 192.168.99.1,
(default) DBG | NetworkMask: 255.255.255.0,
(default) DBG | IPV6Address: fe80::f08c:4842:d7b0:8760,
(default) DBG | IPV6NetworkMaskPrefixLength: 64,
(default) DBG | HardwareAddress: 0a:00:27:00:00:10,
(default) DBG | MediumType: Ethernet,
(default) DBG | Wireless: No,
(default) DBG | Status: Up,
(default) DBG | VBoxNetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter #2,
(default) DBG | ,
(default) DBG | Name: VirtualBox Host-Only Ethernet Adapter,
(default) DBG | GUID: f29686ae-06ce-46c1-aa63-53139bf06fa3,
(default) DBG | DHCP: Disabled,
(default) DBG | IPAddress: 192.168.56.1,
(default) DBG | NetworkMask: 255.255.255.0,
(default) DBG | IPV6Address: fe80::d42b:57f4:8a96:a072,
(default) DBG | IPV6NetworkMaskPrefixLength: 64,
(default) DBG | HardwareAddress: 0a:00:27:00:00:11,
(default) DBG | MediumType: Ethernet,
(default) DBG | Wireless: No,
(default) DBG | Status: Up,
(default) DBG | VBoxNetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter,
(default) DBG | ,
(default) DBG | },
(default) DBG | STDERR:,
(default) DBG | {,
(default) DBG | },
(default) DBG | Adding/Modifying DHCP server "192.168.99.8" with address range "192.168.99.100" - "192.168.99.254"...,
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe list dhcpservers,
(default) DBG | STDOUT:,
(default) DBG | {,
(default) DBG | NetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter,
(default) DBG | IP: 192.168.56.100,
(default) DBG | NetworkMask: 255.255.255.0,
(default) DBG | lowerIPAddress: 192.168.56.101,
(default) DBG | upperIPAddress: 192.168.56.254,
(default) DBG | Enabled: Yes,
(default) DBG | ,
(default) DBG | NetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter #2,
(default) DBG | IP: 192.168.99.12,
(default) DBG | NetworkMask: 255.255.255.0,
(default) DBG | lowerIPAddress: 192.168.99.100,
(default) DBG | upperIPAddress: 192.168.99.254,
(default) DBG | Enabled: Yes,
(default) DBG | ,
(default) DBG | },
(default) DBG | STDERR:,
(default) DBG | {,
(default) DBG | },
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe dhcpserver modify --netname HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter #2 --ip 192.168.99.8 --netmask 255.255.255.0 --lowerip 192.168.99.100 --upperip 192.168.99.254 --enable,
(default) DBG | STDOUT:,
(default) DBG | {,
(default) DBG | },
(default) DBG | STDERR:,
(default) DBG | {,
(default) DBG | },
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm default --nic2 hostonly --nictype2 82540EM --nicpromisc2 deny --hostonlyadapter2 VirtualBox Host-Only Ethernet Adapter #2 --cableconnected2 on,
(default) DBG | STDOUT:,
(default) DBG | {,
(default) DBG | },
(default) DBG | STDERR:,
(default) DBG | {,
(default) DBG | },
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm default --natpf1 delete ssh,
(default) DBG | STDOUT:,
(default) DBG | {,
(default) DBG | },
(default) DBG | STDERR:,
(default) DBG | {,
(default) DBG | },
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm default --natpf1 ssh,
tcp,
127.0.0.1,
58634,
,
22,
(default) DBG | STDOUT:,
(default) DBG | {,
(default) DBG | },
(default) DBG | STDERR:,
(default) DBG | {,
(default) DBG | },
(default) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe startvm default --type headless,
(default) DBG | STDOUT:,
(default) DBG | {,
(default) DBG | Waiting for VM "default" to power on...,
(default) DBG | },
(default) DBG | STDERR:,
(default) DBG | {,
(default) DBG | VBoxManage.exe: error: The virtual machine 'default' has terminated unexpectedly during startup with exit code 1 (0x1). More details may be available in 'C:\Users\rillig\.docker\machine\machines\default\default\Logs\VBoxHardening.log',
(default) DBG | VBoxManage.exe: error: Details: code E_FAIL (0x80004005), component MachineWrap, interface IMachine,
(default) DBG | },
(default) DBG | Checking vm logs: C:\Users\rillig\.docker\machine\machines\default\default\Logs\VBox.log,
Unable to start the VM: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe startvm default --type headless failed:,
VBoxManage.exe: error: The virtual machine 'default' has terminated unexpectedly during startup with exit code 1 (0x1). More details may be available in 'C:\Users\rillig\.docker\machine\machines\default\default\Logs\VBoxHardening.log' ,
VBoxManage.exe: error: Details: code E_FAIL (0x80004005), component MachineWrap, interface IMachine ,
,
Details: 00:01:02.668632 VMMDev: Guest Log: 00:00:10.061627 vminfo Error: Unable to connect to system D-Bus (3/3): D-Bus not installed,
```

**Comment by rillig** [comment]
I finally solved this problem by installing VirtualBox 5.2.8 another time. It would be nice to mention this possible solution in the error message.

**Comment by FrenchBen** [comment]
@rillig unfortunately the error isnt easily traceable to a certain version. We always encourage the version to match that provided in Docker Toolbox, but an upgrade/patch is sometimes needed. Thanks for the feedback
